components/molecule/photoUploader/src/index.scss

```diff
@@ -0,0 +1,310 @@
+$h-photo-uploader-thumb-image-desktop: (
+    $w-photo-uploader-thumb-image-desktop * 3
+  ) / 4;
+$w-photo-uploader-skeleton: $w-photo-uploader-thumb-image;
+$w-photo-uploader-skeleton-desktop: $w-photo-uploader-thumb-image-desktop;
+$maw-photo-uploader-thumb-drag-text: 200px;
+
+#{$base-class} {
+  &-dropzone {
+    background: $bg-photo-uploader;
+    border-radius: $bdrs-l;
+    color: $c-photo-uploader;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    min-height: $mih-photo-uploader;
+    outline: none;
+    padding: $p-photo-uploader;
+    position: relative;
+
+    @include media-breakpoint-up(m) {
+      padding: $p-photo-uploader-desktop;
+    }
+
+    &--disabled {
+      cursor: no-drop;
+    }
+  }
+  &-initialState {
+    align-items: center;
+    border: $bd-photo-uploader;
+    border-radius: $bdrs-l;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    line-height: $lh-m;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &-textState {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: $m-m;
+  }
+
+  &-dragState {
+    align-items: center;
+    border-radius: $bdrs-l;
+    color: $c-photo-uploader-drag-photo;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+    left: 0;
+    min-height: $mih-photo-uploader;
+    position: absolute;
+    top: 0;
+    width: 100%;
+
+    &-textState {
+      line-height: $lh-m;
+      max-width: $maw-photo-uploader-thumb-drag-text;
+      text-align: center;
+    }
+  }
+
+  &-dragState--accepted {
+    background-color: $bg-photo-uploader-drag-photo;
+    border: $bd-photo-uploader;
+  }
+
+  &-dragState--rejected {
+    background-color: $bg-photo-uploader-drag-photo-rejected;
+    border: $bd-photo-uploader-drag-photo-rejected;
+    color: $c-photo-uploader-drag-photo-rejected;
+    z-index: $z-notifications;
+  }
+
+  &-preview-fallback {
+    border-radius: $bdrs-rounded;
+  }
+
+  &-preview {
+    display: grid;
+    grid-auto-rows: 1fr;
+    grid-gap: $m-m;
+    grid-template-columns: repeat(auto-fit, 45%);
+    justify-content: $jc-photo-uploader;
+    margin: 0;
+    padding: 0;
+
+    @include media-breakpoint-up(m) {
+      grid-template-columns: repeat(
+        auto-fit,
+        $w-photo-uploader-thumb-image-desktop
+      );
+    }
+  }
+
+  &-notification {
+    margin-top: $m-m;
+  }
+}
+
+#{$thumb-class} {
+  border: $bd-photo-uploader-thumb;
+  border-radius: $bdrs-l;
+  box-sizing: content-box;
+  cursor: grab;
+  display: inline-block;
+  overflow: hidden;
+
+  &Card {
+    background: $bg-photo-uploader-thumb-card;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    overflow: hidden;
+    position: relative;
+
+    height: 100%;
+
+    &-counter {
+      align-items: center;
+      background-color: $c-primary;
+      border-radius: 999px;
```

Comment:
is it supposed to be rounded? wouldn't `border-radius: 50%` work?

Reply:
there is a token for this https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss#L7

Reply:
I think it's ok, it has to be circular (full rounded corners) not an oval (50%)